.circleci/config.yml

```diff
@@ -240,7 +240,7 @@ jobs:
       # Cache the venv directory that contains dependencies
       - restore_cache:
           keys:
-            - cache-key-cpu-py38-386-{{ checksum "requirements-test.txt" }}-{{ checksum ".circleci/config.yml" }}
+            - cache-key-cpu-py38-{{ checksum "requirements-test.txt" }}-{{ checksum ".circleci/config.yml" }}
```

Comment:
I saw that on another branch, looks like the cached version is "poisoned" (installing torch nightly failed) and we keep pulling it. Ideally we should catch that, but this unlocks CI (generate a new cache, and -386 was an old reference anyway)